benchmarks/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const BenchTable = require('benchtable');
+const _ = require('lodash');
+const suite = new BenchTable('registry');
+
+suite
+	.addInput('published', setup(require('prom-client')))
+	.addInput('local', setup(require('..')))
+	.addFunction('getMetricsAsJSON', registry => {
+		registry.getMetricsAsJSON();
+	})
+	.addFunction('metrics', registry => {
+		registry.metrics();
+	})
+	.on('cycle', event => {
+		if (!event.target.error) {
+			// eslint-disable-next-line no-console
+			console.log(event.target.toString());
+		}
+	})
+	.on('complete', () => {
+		// eslint-disable-next-line no-console
```

Comment:
Instead of this, just add it as exception here: https://github.com/siimon/prom-client/blob/25255c39b5bb225e89e77e69d506061bbb7c4af0/.eslintrc#L64-L69